src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/PrisonApiClientTest.kt

```diff
@@ -455,4 +455,11 @@ class PrisonApiClientTest {
     )
     assertThat(response.offenderNo).isEqualTo(offenderNumber)
   }
+
+  @Test
+  fun `Confirm get Agency Test`() {
+    val agencyId = "AGE1"
+    mockServer.stubGetAgencySuccess(agencyId = "AGE1")
```

Comment:
out of interest, why not call it with 
`stubGetAgencySuccess(agencyId)`
do you have to specify the value again?